core/controllers/access_validators.py

```diff
@@ -143,8 +142,6 @@ class AccountDeletionIsEnabledValidationHandler(base.BaseHandler):
     @acl_decorators.open_access # type: ignore[misc]
     def get(self) -> None:
```

Comment:
Should I remove this get?

Reply:
You should remove this whole class and also the frontend code that calls this class.

Reply:
So does this mean I can remove 
https://github.com/oppia/oppia/blob/develop/core/templates/pages/pending-account-deletion-page/pending-account-deletion-page-root.component.ts
 and
https://github.com/oppia/oppia/blob/develop/core/templates/pages/pending-account-deletion-page/pending-account-deletion-page-root.component.spec.ts

Reply:
I mean all files in 
https://github.com/oppia/oppia/blob/develop/core/templates/pages/pending-account-deletion-page/

Reply:
@vojtechjelinek So if this class is removed do we need files in:
https://github.com/oppia/oppia/tree/develop/core/templates/pages/pending-account-deletion-page

Reply:
They depend on
```python
ACCOUNT_DELETION_IS_ENABLED = (
    '/access_validation_handler/account_deletion_is_enabled');
```
which should be removed after removing this class.

Reply:
Yeah, we need the files, we just need to remove the dependency on the validation handler

Reply:
I have removed the dependency now and related functions and tests now.